draft-cardwell-ccwg-bbr.xml

```diff
@@ -1909,7 +1900,7 @@ following situations:</t>
 </section>
 <section title="Connections without SACK support" anchor="connections-without-sack-support">
 
-<t>If the transport connection does not use SACK (i.e., either or both ends of the connections do not accept SACK), then this algorithm can be extended to estimate approximate delivery rates using duplicate ACKs (much like Reno and <xref target="RFC5681"/> estimates that each duplicate ACK indicates that a data packet has been delivered). The details of this extension will be described in a future version of this draft.</t>
+<t>If the transport connection does not use SACK (i.e., either or both ends of the connections do not accept SACK), then this algorithm can be extended to estimate approximate delivery rates using duplicate ACKs (much like Reno and <xref target="RFC5681"/> estimates that each duplicate ACK indicates that a data packet has been delivered).</t>
```

Comment:
This is TCP centric, FYI.  Also, I don't think we care about the performance of BBR without SACK?